### PR TITLE
bugherd task #524

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

**Comment by rssag** [comment]
this was mainly because of http->https, but i noticed the changes you applied to 10.15.0 and thought some general clean-up could also be applied to 10.14.0 and lower @Oezge-Akin. Please let me know if something only works out in 10.15.0 and higher.

**Review by BeateRixen — APPROVED** [review]

**Comment by Oezge-Akin** [comment]
> this was mainly because of http->https, but i noticed the changes you applied to 10.15.0 and thought some general clean-up could also be applied to 10.14.0 and lower @Oezge-Akin. Please let me know if something only works out in 10.15.0 and higher.

Yes, I changed them with this https://github.com/SoftwareAG/c8y-docs/pull/589/files. They also apply to prev. GA versions. If you want you can directly cherry-pick them, there are few more things there.

File: content/device-sdk/device-sdk-rest-bundle/device-integration.md [hunk 1]
```diff
@@ -261,7 +261,9 @@ For example, assume a child device with the URL "https://.../inventory/managedOb
     POST /inventory/managedObjects/2480300/childDevices HTTP/1.1
     Content-Type: application/vnd.com.nsn.cumulocity.managedobjectreference+json
     {
-    "managedObject" : { "self" : "https://.../inventory/managedObjects/2543801" }
+    "managedObject": {
```
Comment (Oezge-Akin) [inline content/device-sdk/device-sdk-rest-bundle/device-integration.md:264]: "managedObject" : {
            "id" : "2543801"
        }

**Review by MWindel — APPROVED** [review]

File: content/device-sdk/device-sdk-rest-bundle/device-integration.md (hunk 1 above)
Comment (rssag) [reply content/device-sdk/device-sdk-rest-bundle/device-integration.md]: done

**Comment by rssag** [comment]
> > this was mainly because of http->https, but i noticed the changes you applied to 10.15.0 and thought some general clean-up could also be applied to 10.14.0 and lower @Oezge-Akin. Please let me know if something only works out in 10.15.0 and higher.
> 
> Yes, I changed them with this https://github.com/SoftwareAG/c8y-docs/pull/589/files. They also apply to prev. GA versions. If you want you can directly cherry-pick them, there are few more things there.

there's a cherry-pick conflict - i'll use this PR instead. i think the https vs http is most important. thanks :)

**Comment by rssag** [comment]
fixed in 10.14.0, 10.13.0 and 10.11.0.

**Pull request merged**